xAPI.md

```diff
@@ -2020,6 +2020,21 @@ contain attachments.
 * The Client MAY send batches of type "application/json" where every attachment
 Object has a fileUrl, ignoring all requirements based on the "multipart/mixed" format.
 
+###### File URL
+The File URL is intended to provide a location from which the LRS or another system can retrieve the
+attachment. There are, however, no requirements for the owner of the attachment to make the 
+attachment data available at the location indefinately or to make the attachment publically
+available without security restrictions. When determining attachment hosting arrangements, 
+designers of systems that will send Statements using the "fileUrl" property are encouraged to 
+consider the needs of end recipient(s) of the Statement especially if the attachment content 
+is not included with the Statement.
+
+* The attachment data SHOULD be retirevable at the IRL indicated by the fileUrl.
```

Comment:
What is the actual workflow here?  An AP provides a File URL and the LRS just passes it along, right?  We don't (and I'm not suggesting we should) have any concept of an LRS internalizing the file and providing its own file URL for use when this Statement were returned, and I don't think we should disallow this behavior either. 

I think the current language is fine as usage will vary and the actual storage/retrieval of the files themselves is beyond the scope of the spec.